pkgs/tools/networking/network-manager/applet/default.nix

```diff
@@ -21,16 +20,15 @@
 , python3
 , gtk3
 , libappindicator-gtk3
-, glib
```

Comment:
The code does in fact depend on GLib:

https://gitlab.gnome.org/GNOME/network-manager-applet/-/blob/74a71cbe52c961701d4d8b70bcf76fa0ed83a5d3/meson.build#L107

(It gets it from GTK propagated build inputs but we generally prefer listing direct dependencies explicitly.)

Reply:
Ack, thanks for noting.

Reply:
So, it should have it listed in `buildInputs` as well, then?

Reply:
Yup.